pom.xml

```diff
@@ -400,6 +400,11 @@
 			<version>1.5.3</version>
 			<scope>test</scope>
 		</dependency>
+		 <dependency>
```

Comment:
Pom shoudl be sorted according to Maven recommendations. You can use `mvn sortpom:sort` to sort it automatically (https://code.google.com/p/sortpom/).